RELEASE-NOTES.txt

```diff
@@ -1,5 +1,8 @@
 Unreleased
 ---
+
+1.69.0
```

Comment:
I noticed that the entry "Fix app freeze when closing link picker while virtual keyboard is hidden" has been included in `1.69.0` but [the PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4443) was merged after the release was cut ([here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4438/files?authenticity_token=oDLzV1onSLSEyk638jPZOgjByqbOLDtLeJJPLELWhIerXgUhCwT2CPWcmgTuz7mnPvyrmVyiweIdvwom30k_xg&hide-deleted-files=true&file-filters%5B%5D=#diff-558ce8820a544da19f7fccdb2d1d661e46d68c130a1a39abdd222aa3484a5873) you can check the release notes of the release branch). I'll update the release notes to reflect this change.

Reply:
Similarly, we could also update the `react-native-editor` changelog file in Gutenberg, which also included this entry under `1.69.0`.

Reply:
I'm thinking that since this change will be released as a betafix, I think it makes more sense to update both release notes and changelog files there.

Reply:
Good catch Carlos! The good thing is, this was _not_ included in the actual release (nor release notes), but was only included in this merge back to `develop` (as an auto-resolved conflict when merging `develop` in [this commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4448/commits/dcae8cfacd50f126fa13dc9a6712b32d38c09223)).

> I'm thinking that since this change will be released as a betafix, I think it makes more sense to update both release notes and changelog files there.

I agree, that's the correct approach, and actually, can be under it's own section `1.69.1`. Thanks for pointing this out!